flow/twilio/flex/status.yaml

```diff
@@ -3,26 +3,34 @@ triggers:
   - event_type: meya.twilio.flex.event.reservation_accepted
 
 steps:
-  - status: Agent (@ flow.event.data.agent_name ) entered chat
+  - status: (@ flow.event.data.agent_name ) has joined the chat.
+  - thread_set:
+      agent_name: (@ flow.event.data.agent_name )
 ---
 id: flow.twilio.flex.reservation_completed
 triggers:
   - event_type: meya.twilio.flex.event.reservation_completed
 
 steps:
-  - status: The agent ended the chat
+  - status: (@ thread.agent_name ) has left the chat.
```

Comment:
Not required, but I wonder if we have access to the agent name in the event rather than using `thread` bookkeeping for this use-case.